jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/Dumpable.java

```diff
@@ -50,24 +56,56 @@ default String dump()
     void dump(Appendable out, String indent) throws IOException;
 
     /**
-     * Utility method to implement {@link #dump()} by calling {@link #dump(Appendable, String)}
+     * Utility method to call dump to a {@link String}
      *
      * @param dumpable The dumpable to dump
      * @return The dumped string
+     * @see #dump(Appendable, String)
      */
     static String dump(Dumpable dumpable)
     {
         StringBuilder b = new StringBuilder();
+        dump(dumpable, b);
+        return b.toString();
+    }
+
+    DateTimeFormatter UTC_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").withZone(ZoneOffset.UTC);
+    DateTimeFormatter LOCAL_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS Z").withZone(ZoneId.of(System.getProperty("user.timezone")));
+
+    /**
+     * Utility method to dump to an {@link Appendable}
+     *
+     * @param dumpable The dumpable to dump
+     * @param out The destination of the dump
+     */
+    static void dump(Dumpable dumpable, Appendable out)
+    {
         try
         {
-            dumpable.dump(b, "");
+            dumpable.dump(out, "");
+
+            out.append(KEY);
+            Runtime runtime = Runtime.getRuntime();
+            Instant now = Instant.now();
+            out.append("JVM: %s %s; OS: %s %s %s; Jetty: %s; CPUs: %d; mem(free/total/max): %,d/%,d/%,d MB\nUTC: %s; %s: %s".formatted(
+                System.getProperty("java.runtime.name"),
+                System.getProperty("java.runtime.version"),
```

Comment:
The vendor is missing, and it's important (say IBM).
I would use: `java.vendor` `java.vm.name` `java.vm.version`.

For example, IBM's does not have any `java.runtime.*` properties (at least according to their docs).

Use `MiB` instead of `MB` (powers of 1024 instead of 1000).

I would just print the UTC time, not the local one.

Reply:
I like having UTC and local time... it is OK for you who only lives an hour or 2 off UTC, but for those of us that struggle with bigger differences, it can be nice to have them both listed :)
Having said that, my laptop still thinks it is in Italy??!?!?